charts/flagsmith/README.md

```diff
@@ -1,10 +1,127 @@
 # Flagsmith Helm chart
 
-## Requirements
+## Installation
+
+```bash
+helm repo add flagsmith https://flagsmith.github.io/flagsmith-charts/
+helm install flagsmith flagsmith/flagsmith
+```
+
+This will install using default options. (Pods may error and restart
+while waiting for other pods to become ready.).
+
+### Ingress configuration
+
+The above will start pods in the cluster, but to gain web access to
+Flagsmith, do one of the following:
+
+#### In a cluster that has an ingress controller
+
+Set the following values for flagsmith, with changes as needed to
+accommodate your ingress controller, and any associated DNS
+changes. Also, set the `API_URL` env-var such that the URL is
+reachable from a browser accessing the frontend.
+
+Eg:
+
+```yaml
+ingress:
+  frontend:
+    enabled: true
+    hosts:
+      - host: flagsmith.[MYDOMAIN]
+        paths:
+          - /
+  api:
+    enabled: true
+    hosts:
+      - host: flagsmith.[MYDOMAIN]
+        paths:
+          - /api/
```

Comment:
there's also a /heath endpoint so might be worth just putting in the root path here? Unless there's a reason not to?

Reply:
@dabeeeenster What might need to access `/health`? If it is just an "implementation detail", eg for the k8s healthchecks, then we don't need to expose it. If a user's browser needs to access it, then we would. Or, I suppose, for uptime monitoring, maybe?

Reply:
It is useful for it to be accessible outside of k8s for latency checking etc too so yeah it would be good to have it

Reply:
OK, added. I've made a note to revisit some of this to make it harder for someone to misconfigure.